test/test_optimize.py

```diff
@@ -142,17 +144,15 @@ def test_rosenbrock(self):
         self.assertAlmostEqual(mle.optimized_params_pd['x'][0], 1, places=3)
         self.assertAlmostEqual(mle.optimized_params_dict['x'], 1, places=3)
 
-        self.assertEqual(
-            mle.stan_variable('x', inc_iterations=True).shape, (36,)
-        )
-        self.assertEqual(mle.optimized_iterations_np.shape, (36, 3))
+        last_iter = mle.optimized_iterations_np.shape[0] - 1
```

Comment:
since `optimize` function hasn't changed in forever, it should take 36 iterations to find MLE.
but really not sure about what the algorithm does to know whether or not it should always take exactly 36 iterations, given seed and all other initialization.